apps/web/styles/globals.css

```diff
@@ -2,19 +2,99 @@
 @tailwind components;
 @tailwind utilities;
 
-
+:root {
```

Comment:
Shall we add `color-scheme: dark;` here? That makes the scrollbars default dark instead of light as well, looks a lot better I think. 

See for example the text area scrollbars:

before:
![CleanShot 2023-04-05 at 11 51 53@2x](https://user-images.githubusercontent.com/2969573/230046459-17d5e6ca-a3c0-4dbf-ab98-94126b1185e6.jpg)

after:
![CleanShot 2023-04-05 at 11 52 22@2x](https://user-images.githubusercontent.com/2969573/230046562-f939b103-2dda-4a83-9b41-b837a64d8892.jpg)


Both cases are ugly because of cut offs etc, but I think the automatic darker scrollbar already improves it by a lot.

Reply:
Pushed the change.